Beyond08-SOW.md

```diff
@@ -0,0 +1,467 @@
+       of your project.
+    4. Getting started - How to set up a development environment to begin
+       working on the project code.
+
+* Architecture guide
+
+    1. How to understand the project code - which major frameworks are used, the
+       class structure, technical design background information and where to
+       find the different kinds of files in the project
+
+* License
+
+    1. A License file defines copyright protections around your code and
+       what users and contributors are legally allowed to do.
+    2. In the open source community, developers typically pick an
+       [OSI-approved](https://opensource.org/licenses) license for their  
+       projects.
+    3. In Beyond, the mentors will create the project repo for you with the 
+       permissive [MIT license](https://opensource.org/licenses/MIT). We require
+       that you keep this license for the Beyond project in order to facilitate
+       sharing with other Beyond students and the Red Hat team.
+
+## System Requirements
+
+* Computer
+* 8 GB RAM
+* OS - Windows, OSX, Linux
+* Language - English (Username should be in English)
+* Installations:
+
+    * [Git](https://git-scm.com/downloads)
+    * [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+    * [Vagrant](https://www.vagrantup.com/downloads.html)
+
+## List of things you can do to improve communications with your instructors and teammates and your overall "visibility"
+
+* Set up a recognizable personal avatar (profile picture) on both GitHub and
+  Slack.
+* Ask for code reviews by mentioning the mentors in the PR
+* Mention people with "@" in GitHub issues and PRs when you want them to comment
+  or review.
+* Link your college email address with your GitHub account, so we can find you by
+  the address
+
+## Buzz words
+
+* PR - A pull request (PR) is a method of submitting contributions to an open
+  development project.
+* CR - code review is a software quality assurance activity in which one or
+  several people check a program mainly by viewing and reading parts of its
+  source code. At least one of the persons must not be the code's author.
+* LGTM - looks good to me. Commonly used in CR.
+* Client-Server architecture - a computing model in which the server hosts,
+  delivers and manages most of the resources and services to be consumed by the
+  client. This type of architecture has one or more client computers connected
+  to a central server.
+
+* Framework - a code library that makes a developer's life easier when building
+  reliable, scalable, and maintainable web applications by providing reusable
+  code or extensions for common operations.
+* [Retrospective](https://geekbot.com/blog/what-is-a-sprint-retrospective-and-how-to-run-it)
+* [Daily stand up](https://geekbot.com/blog/daily-standup-meeting)
+* [Technical debt](https://en.wikipedia.org/wiki/Technical_debt)
+* [Refactoring](https://refactoring.com)
+
```

Comment:
maybe we can add a table/list of links to the latest student projects 
to let the students realize what we are talking about

Reply:
Agree